src/main/java/uk/gov/hmcts/ccd/domain/model/definition/CaseField.java

```diff
@@ -140,6 +144,46 @@ public void propagateACLsToNestedFields() {
         propagateACLsToNestedFields(this, this.accessControlLists);
     }
 
+    @JsonIgnore
+    public FieldType getFieldTypeByPath(String path) {
+        if (StringUtils.isBlank(path)) {
+            return this.getFieldType();
+        } else {
+            CaseField caseFieldByPath = this.findNestedElementByPath(path);
+            return caseFieldByPath.getFieldType();
+        }
+    }
+
+    @JsonIgnore
+    public CaseField findNestedElementByPath(String path) {
```

Comment:
can this be private? In that case we could remove the blank field validation because we already did it in `getFieldTypeByPath`

Reply:
Now used by CaseType getCaseFieldByPath(...) method